src/components/App.tsx

```diff
@@ -95,6 +81,13 @@ const App: React.FC = () => {
       })
     );
   };
+  const FocusInput: React.FocusEventHandler<HTMLInputElement> = () => {
+    return setCheckPlus((checkPlus = "1"));
```

Comment:
Incorrect way of using useState. Please read react docs

Reply:
Done